src/Microsoft.AspNet.Mvc.Razor/DefaultTagHelperActivator.cs

```diff
@@ -48,6 +48,10 @@ private static PropertyActivator<ViewContext> CreateActivateInfo(PropertyInfo pr
             {
                 valueAccessor = viewContext => viewContext;
             }
+            else if(property.PropertyType == typeof(ViewDataDictionary))
```

Comment:
nit: `property.PropertyType` now used three times in this method.  might be easier to read with a `propertyType` variable.